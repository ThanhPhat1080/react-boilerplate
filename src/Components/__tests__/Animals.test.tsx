// Libraries
import * as renderer from 'react-test-renderer';

// Components
import { Animals } from '@Components';

describe('Testing Animals Component', () => {
  test('It should render correctly', () => {
    const component = renderer.create(<Animals />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
