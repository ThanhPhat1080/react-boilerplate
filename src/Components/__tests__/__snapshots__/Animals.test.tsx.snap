// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Animals Component It should render correctly 1`] = `
<div>
  We have
   
  <span
    aria-label="list animals"
    role="img"
  >
    🐕 🦊 🐒 🦍 🦥 🦒
  </span>
</div>
`;
